### PR TITLE
Adding loading bar to pdf viewer

### PR DESCRIPTION
**Pull request by ChrisKraljevicD2L**

### CODE REVIEW

File: d2l-pdf-viewer.html [hunk 1]
```diff
@@ -102,13 +104,17 @@
...
+				this.$.loadingBar.start();
+
 				pdfjsLib.getDocument({
 					url: this.src
 				}).then(pdfDocument => {
 					this._pdfViewer.setDocument(pdfDocument);
 					this._pagesCount = pdfDocument.numPages;
 					this._pageNumber = this._pdfViewer.currentPageNumber;
 
+					this.$.loadingBar.finish();
```
Comment (apalaniuk) [inline d2l-pdf-viewer.html]: This or 'pagesinit'?

Comment (ChrisKraljevicD2L) [reply d2l-pdf-viewer.html]: Hmm, good point. I'll put it in pagesinit.

File: d2l-pdf-viewer.html [hunk 2]
```diff
@@ -102,6 +104,8 @@
 				// Add event listeners before loading document
 				this._addEventListeners();
 
+				this.$.loadingBar.start();
+
 				pdfjsLib.getDocument({
 					url: this.src
```
Comment (awikkerink) [inline d2l-pdf-viewer.html]: This is unrelated, but this logic should be under the `attached` callback, or better yet when the value of `src` changes.

File: d2l-pdf-viewer.html [hunk 3]
```diff
@@ -152,9 +156,13 @@
 					}, 100);
 				}
 			},
+			_srcChanged: function() {
+				this.$.loadingBar.start();
```
Comment (apalaniuk) [inline d2l-pdf-viewer.html]: Probably makes more sense to do the load-on-property-change in a separate PR, as right now this is a bit divorced from the actual PDF loading lifecycle.

**Comment by ChrisKraljevicD2L** [comment]
Whoops, wrong PR for that commit

**Pull request merged**